commands/Tools/twitch.js

```diff
@@ -0,0 +1,69 @@
+exports.run = async(client, msg, [twitchName]) => {
+    const rp = require("request-promise-native");
```

Comment:
And in an attempt to keep dependencies down, can you switch this so it uses `snekfetch`